src/v3/src/components/Widget/index.tsx

```diff
@@ -100,16 +102,13 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   const [loginHint, setloginHint] = useState<string | null>(null);
   const brandedTheme = mapMuiThemeFromBrand(brandColors, muiThemeOverrides);
 
-  useEffect(() => {
-    // If we need to load a language (or apply custom i18n overrides), do
-    // this now and re-run render after it's finished.
+  const initLanguage = useCallback(async () => {
```

Comment:
Had to convert to a function because there was intermittent delays between when the widget loaded and when the requested language was available.